bindings/experimental/distrdf/python/DistRDF/ComputationGraphGenerator.py

```diff
@@ -157,7 +158,8 @@ def generate_computation_graph(self, previous_node, range_id, distrdf_node=None)
             # we just retrieved
             previous_node = pyroot_node
 
-            if (operation.is_action() or operation.is_instant_action()):
+            if isinstance(operation, (Action, InstantAction)):
```

Comment:
I'm wondering: do we ever need to know if it is an InstantAction but not an Action?

Reply:
Mmm, yes in the proxy :)

Reply:
Exactly, plus in another PR we might decide to further have an `InstantAction` subclass for Snapshot and another for AsNumpy if we want to use the `singledispatch` pattern also in places like [here](https://github.com/root-project/root/blob/deb76c6de6e43cf52a04fa9ab62d0c06a99b392e/bindings/experimental/distrdf/python/DistRDF/ComputationGraphGenerator.py#L73)